basil/TL/Visa.py

```diff
@@ -32,10 +34,13 @@ def init(self):
         backend = self._init.get('backend', '')  # Empty string means std. backend (NI VISA)
         rm = visa.ResourceManager(backend)
         try:
-            logger.info('BASIL VISA TL with %s backend found the following devices: %s', backend, ", ".join(rm.list_resources()))
+            logger.info('BASIL VISA TL with %s backend found the following devices: %s', backend,
+                        ", ".join(rm.list_resources()))
```

Comment:
this change is due to pep8 E501 enforcement? what is the policy in basil? i don't find it useful in modern times with widescreen displays everywhere...